docker-entry

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+sleep 10
+
+echo "Starting DB maintenance"
```

Comment:
Please add a check to validate that it can connect to the DB. A simple `select version();` query should do the trick. Exit with an error code and appropriate message if it fails.